src/components/layouts/WideLayout.js

```diff
@@ -14,7 +14,8 @@ const useStyles = makeStyles(theme => ({
   }
 }));
 
-export default function AboutLayout({ children, title }) {
+//This layout is used for pages, where we want to use the whole width of the page.
```

Comment:
I think the intent is pretty clear without this comment, so I would consider removing. Feel free to ignore.